plugins/inputs/prometheus/start.go

```diff
@@ -335,9 +349,34 @@ func reloadConfig(filename string, logger log.Logger, rls ...func(*config.Config
 
 		sc.RelabelConfigs = append(relabelConfigs, sc.RelabelConfigs...)
 		sc.MetricRelabelConfigs = append(metricNameRelabelConfigs, sc.MetricRelabelConfigs...)
-
 	}
+}
+func reloadConfig(filename string, logger log.Logger, taManager *TargetAllocatorManager, rls ...func(*config.Config) error) (err error) {
```

Comment:
nit: function spacing
```suggestion
}

func reloadConfig(filename string, logger log.Logger, taManager *TargetAllocatorManager, rls ...func(*config.Config) error) (err error) {
```